buf.gen.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Nordix
```

Comment:
Should we use the same copyright notice we use for other CDEvents projects?
e.g. https://github.com/cdevents/sdk-go/blob/39341b23818a6a352fea15ebbb79a0bdf91d4dd0/pkg/api/bindings.go#L1-L17

Reply:
I think it is a legal entity for us to work through Nordix and using the Copyright also with Nordix.
Any comments @andrew-est on using Nordix Copyright

Reply:
I'm not a legal expert but I think the copyright is from the person/group giving the contribution and as such when we make it it comes from Nordix, thats at least what we have done with other CNCF projects, but if its an issue we can check it and come back